src/components/Picker/BasePicker/basePickerPropTypes.js

```diff
@@ -11,8 +11,8 @@ const propTypes = {
     /** Whether or not to show the disabled styles */
     disabled: PropTypes.bool,
 
-    /** Should the picker be styled for errors  */
-    hasError: PropTypes.bool,
+    /** Error text to display */
+    errorText: PropTypes.string,
```

Comment:
This is still being used in a few places, e.g. [here](https://github.com/Expensify/App/blob/e11e3627d5c8f345c29f7cb470190a58df0afc54/src/pages/ReimbursementAccount/CompanyStep.js#L273). So prematurely removing it will throw errors.

Reply:
Oh my, I didn't know. I though I forgot to remove it. Fixing it.

Reply:
That means that I need to declare both `errorText` and `hasError`  on `basePickerPropTypes.js`. Also I need to integrate `this.props.hasError` to error style logic (currently it is only usign `this.props.errorText` to style).

Reply:
Also, do you think we should add back `hasError` prop on `Picker`'s PropTypes if we are still using it somewhere?